exchanges/bybit/bybit_websocket.go

```diff
@@ -167,28 +167,30 @@ func (by *Bybit) handleSubscriptions(operation string, subs subscription.List) (
 	if err != nil {
 		return
 	}
-	chans := []string{}
-	authChans := []string{}
+	var chans subscription.List
+	var authChans subscription.List
```

Comment:
Nit:
If `handleSubscriptions` accepts `auth bool` then you can call it twice with `subs.Public(), false` and `subs.Private(), true`.
Then all the code below becomes the same, and auth field uses the method param.
Side effect is that authed subs would get batched into groups of 10, but I think that's neither an issue or possible.